api-docs/api-reference/methods/roles/put-grant-roles-to-user-v2.0.rst

```diff
@@ -0,0 +1,152 @@
+.. _put-grant-roles-to-user-v2.0:
+
+Grant roles to a user
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+   PUT /v2.0/users/{userId}/RAX-AUTH/roles
+
+Grant roles to a provisioned user. Existing roles set on the user that are not
+being modified remain unchanged.
+
+.. note::
+
+   - The caller must have at least the ``identity:user-manage`` role. If the
+     caller has only domain access, verify that the caller's domain matches
+     the target user's domain.
+   - The tenants on assignments must belong to the same domain as the target
+     user's domain.
+   - The role assignments cannot assign identity user type roles with the
+     exception of ``identity:user-manage``.
+   - The caller must be authorized to assign the specified roles.
+   - The tenant assignments can include either ["*"] or a set of tenant ids.
+   - This request effectively replaces any assignment that already exists for
+     these roles on the user.
+
```

Comment:
Just a check in here to make sure that you want the italics in the text above.

Reply:
This seem to be github doing some sort of rendering. In the actual documentation it's displayed correctly.